### PR TITLE
Features

### PR DESCRIPTION
**Pull request by silkspace**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -356,15 +356,33 @@ Automatically and intelligently transform text, numbers, booleans, and other for
...
     new_df = pandas.read_csv(...)
     X_new, _ = g.transform(new_df, None, kind='nodes')
     preds = model.predict(X_new)
     ```
 
+ * Encode model definitions and compare models against each other
+
+   ```python
+    # graphistry
+    from graphistry.features import search_model, topic_model, ngrams_model, ModelDict, default_parameters
```
Comment (lmeyerov) [inline README.md:372]: here and docstr, put a link to where to find models, e.g., hf?

File: README.md [hunk 2]
```diff
@@ -356,15 +356,33 @@ Automatically and intelligently transform text, numbers, booleans, and other for
...
+
+    g = graphistry.nodes(df)
+    g = g.umap(X=[..], y=[..], **search_model)
+
+    # set custom encoding model
```
Comment (lmeyerov) [inline README.md:377]: swap in a more compelling & real model, like a different language or for translation?

**Review by lmeyerov — CHANGES_REQUESTED** [review]
Code looks good, just docs can be more useful

Maybe there should be an ipynb on a few representative & most likely feature engineering / model tricks? Changing language model, doing translation, ...

File: README.md (hunk 2 above)
Comment (silkspace) [reply README.md:377]: I agree it makes sense, but in this demo the `search_model` (MSMARCO2) is the best model.

Comment (lmeyerov) [reply README.md:377]: yep and maybe also a url for where they can find them

**Pull request closed without merging**